tools/geojson_simplifier/simplify_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the import_data.py script.
+
+    Typical usage:
+    python3 simplify_test.py
+"""
+
+import unittest
+import geojson
+import simplify
+
+TEST_DATA_DIR = "test-data/"
+
+
+class GeojsonSimplifierTest(unittest.TestCase):
+    def test_california(self):
```

Comment:
\# TODO(fpernice-google): Add some simple tests using hard-coded values including the empty input, a single input, and a few inputs.